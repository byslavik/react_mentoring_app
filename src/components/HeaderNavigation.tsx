import * as React from 'react';

import { Link } from 'react-router-dom';
import * as buttons from '../scss/buttons.scss';

export class HeaderNavigation extends React.Component {
  render() {
    return (
      <nav>
          <li><Link to="/search/" className={buttons.buttonWhite}>Search</Link></li>
      </nav>
      );
  }
}