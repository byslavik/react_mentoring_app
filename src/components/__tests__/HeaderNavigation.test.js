import * as React from 'react';
import {HeaderNavigation} from '../HeaderNavigation';
import * as renderer from 'react-test-renderer';

jest.mock('react-router-dom',()=> ({Link:'Link'}));

describe('HeaderNavigation component', ()=>{
  it('HeaderNavigation component shows correctly', () => {
    const tree = renderer.create(
      <HeaderNavigation />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
})