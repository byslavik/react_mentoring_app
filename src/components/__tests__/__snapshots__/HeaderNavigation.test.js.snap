// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderNavigation component HeaderNavigation component shows correctly 1`] = `
<nav>
  <li>
    <Link
      className={undefined}
      to="/search/"
    >
      Search
    </Link>
  </li>
</nav>
`;
